src/applications/vaos/components/FacilityPhone.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Telephone from '@department-of-veterans-affairs/formation-react/Telephone';
+
+export default function FacilityPhone({ contact }) {
+  if (!contact) {
+    return null;
+  }
+
+  const [number, extension] = contact.split('x');
```

Comment:
The Telephone component doesn't handle extensions in the format that the facilities api returns them in, so I made a component to handle that, which is used in two places.